integration_tests/requirements.txt

```diff
@@ -4,4 +4,4 @@ dbt-redshift>=1.3.0,<2.0.0
 dbt-postgres>=1.3.0,<2.0.0
 dbt-spark>=1.3.0,<2.0.0
 dbt-spark[PyHive]>=1.3.0,<2.0.0
-dbt-databricks>=1.3.0,<2.0.0
\ No newline at end of file
+dbt-databricks>=1.6.0,<2.0.0
```

Comment:
Same question about if there is a reason this is different than the other versions

Reply:
Oh no real reason -- I just noticed this was inconsistent with some other packages and updated it accordingly. I can revert this if preferable!

Reply:
No this is okay to keep, I just wanted to make sure there wasn't a specific reason and if we needed to adjust our dbt required version for this update. All good to keep. Thanks!